### DISCUSSION TITLE
Is there a way to redirect user based on their browser language?

### DISCUSSION THREAD

**Discussion by gbreux** [vercel/vercel#4499] [Help] [answered]
Hi,
I just started to use Vercel and I was wondering if there were a way to conditionnaly redirect a user based on their language, just like [Netlify do here](https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file)?

Thanks a lot,
Guillaume

**Comment by coetry** [comment, marked as answer]
Hi @gbreux we currently don't have a platform feature to support that, but are exploring different ways to make that available.

**Reply by stefanprobst** [reply]
is there an issue to subscribe to for updates?

**Reply by paulogdm** [reply]
We will release this with https://github.com/vercel/next.js/discussions/17078

**Comment by euskadi31** [comment]
Hi, Any news ?

**Reply by amyegan** [reply]
Hi @euskadi31! We have added Internationalization support in the time since the original post. Here's some documentation and example pages that might be helpful:
- [Internationalization](https://vercel.com/docs/concepts/next.js/internationalization#)
- [i18n in Edge functions](https://vercel.com/templates/next.js/edge-functions-i18n)
- [Next.js Internationalized Routing](https://nextjs.org/docs/advanced-features/i18n-routing)

**Reply by euskadi31** [reply]
thx, I use Angular...

**Reply by amyegan** [reply]
I can see how the Next.js examples wouldn't be helpful in that case! Angular has it's own [Internationalization and Localization documentation](https://angular.io/guide/i18n-overview) and [i18n example](https://angular.io/guide/i18n-example). That would be the best place to start.

**Reply by layout-ch** [reply]
Yes but there is no documentation anywhere on how to redirect a visitor based on his browser language. And not specially for react but for any fe... Is it not possible with the vercel.json?
